test/Microsoft.Azure.SignalR.Management.Tests/NegotiateProcessorFacts.cs

```diff
@@ -115,9 +118,12 @@ internal async Task GenerateClientEndpointTestWithClientEndpoint()
             {
                 o.ServiceEndpoints = endpoints;
                 o.ServiceTransportType = ServiceTransportType.Persistent;
-            }).BuildServiceProvider();
+            })
+            .AddSingleton(sp => ActivatorUtilities.CreateInstance<NegotiateProcessor>(sp, HubName))
+            .AddSingleton<IEndpointGetter>(sp => ActivatorUtilities.CreateInstance<DefaultEndpointGetter>(sp, HubName))
```

Comment:
add test to show the Getter can specify endpoints without impact the whole service hub context?

Reply:
I think if the default getter doesn't break the current tests, then it's Ok.